### DISCUSSION TITLE
记录一个奇怪的崩溃事件

### DISCUSSION THREAD

**Discussion by LLIce** [kkkgo/PaoPaoDNS#189] [经验分享]
今天起来发现很卡顿，查询adg发现主dns延迟很高，再过一会就崩了，然后看到14号有个更新，更新后还是不对劲。
HTTP_FILE映射网页直接打不开，跑test直接提示dns not ready。
一顿操作docker还是有问题，最后把配置文件全清空突然好了，然后一个个文件排查，最后定位到了force_dnscrypt_list.txt。
这个网页之前我打开有问题，就把他添加进nocn(以前叫这)，以前也没出过这问题，今天突然炸了。
![32657](https://github.com/user-attachments/assets/649d2f48-49f7-47fc-9f2a-89909bfc52f1)

**Comment by kkkgo** [comment]
是domain不是doman

**Reply by LLIce** [reply]
才看到，还真是啊，添加这么久都没发现，搞笑了。
